internal/rpchelp/helpdescs_en_US.go

```diff
@@ -143,7 +143,7 @@ var helpDescsEnUS = map[string]string{
 	"getstakeinforesult-live":             "Number of mature, active tickets owned by this wallet",
 	"getstakeinforesult-proportionlive":   "(Live / PoolSize)",
 	"getstakeinforesult-voted":            "Number of votes cast by this wallet",
-	"getstakeinforesult-totalsubsidy":     "Total amount of coins earned by stake mining",
+	"getstakeinforesult-totalsubsidy":     "Total amount of coins earned by proof-of-stake voting",
```

Comment:
The help text needs to be regenerated after modifying this file.  From `rpc/legacyrpc` run `go generate`.

Reply:
Already done here, am I missing something?

https://github.com/decred/dcrwallet/pull/1371/files#diff-60987c8987b22e2470c08562ec0367cbR31

Reply:
nope, my bad!